Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -72,6 +72,42 @@ public void InlineAutoDataSuppliesDataSpecimensOnlyForNonProvidedValues(string s
             Assert.NotEqual("bar", s3);
         }
 
+        // This test and its associated types is used to document one of the
+        // InlineAutoDataAttribute constructor overloads.
+        [Theory]
+        [MyCustomInlineAutoData(1337)]
+        [MyCustomInlineAutoData(1337, 7)]
```

Comment:
Assigning a `const` requires yet another line of code in the code example. It's also another level of indirection. Would that make the example easier to understand?

Renaming `y` to `dummy` is an option, but does it make the example easier to understand? If we do that, the argument list would be `(int x, int dummy, int z)`. Would the reader wonder what happened to `y` from the range `y..z`?

Would `(int x, int dummyY, int z)` be better?

Or would it be better to entirely remove the unused argument? The reason I included it was to show that you could have an argument that could be provided by a constant, or by AutoFixture.

Another option is to add a comment to the body of the test, like this:

```
Assert.Equal(1337, x);
// y can vary, so we can't express any meaningful assertion for it.
Assert.Equal(42, z);
```